lib/engine/game/g_1856.rb

```diff
@@ -392,13 +453,30 @@ def grant_right(corporation, type)
         ))
       end
 
+      def event_no_more_escrow_corps!
+        @log << 'New corporations will be started as incremental cap corporations'
+        remove_destinations
+      end
+
+      def event_no_more_incremental_corps!
+        @log << 'New corporations will be started as full cap corporations'
+      end
+
       def event_close_companies!
         # The tokens reserved for the company's buyer are sent to the bank if closed before being bought in
         @available_bridge_tokens += 1 if bridge.owned_by_player?
         @available_tunnel_tokens += 1 if tunnel.owned_by_player?
         super
       end
 
+      def remove_destinations
+        @corporations.select { |corp| corp.capitalization != :escrow && corp != national }.each do |corp|
+          corp.destinations.each do |dest|
+            hex_by_id(dest).tile.icons.reject! { |icon| icon.destination? && icon.name == corp.id }
```

Comment:
why do we even need a destination? why can't it just be an icon?

Reply:
Destination as a Part has been removed